dist/bundle.js

```diff
@@ -0,0 +1,743 @@
+function generateTarget(matrix) {
+  var legal = false,
+      par = matrix[0][0] * matrix[1][1] - matrix[0][1] * matrix[1][0];
+  var newX, newY;
+
+  while (!legal) {
+    newX = getRandom(0,500);
+    newY = getRandom(0,500);
+    var pre = screenToMath(newX,newY);
+    var prex = (matrix[1][1] * pre[0] - matrix[0][1] * pre[1]) / par,
+        prey = (- matrix[1][0] * pre[0] + matrix[0][0] * pre[1]) / par;
+    pre = mathToScreen(prex,prey);
+
+    if (pre[0] >= 0 && pre[0] <= 500 && pre[1] >= 0 && pre[1] <= 500) {
+      legal = true;
+      var targetSettings = {
+      	x: newX,
+      	y: newY,
+      	r: 20,
+      	color: "black",
+      	isScore: false
+      };
+      var newTarget = new Target(targetSettings);
+      newTarget.drawTarget();
+    }
+  }
+}
+
+module.exports = Canvas;
+},{"../actors/target.js":1,"../actors/vector.js":2}],4:[function(require,module,exports){
+function startLevel1() {
+
+
+	var inputSettings = {
+	    minX: -10,
+	    minY: -10,
+	    maxX: 10,
+	    maxY: 10,
+	    pixelWidth: 500,
+	    pixelHeight: 500
+	};
+
+	var outputSettings = {
+	    minX: -10,
+	    minY: -10,
+	    maxX: 10,
+	    maxY: 10,
+	    pixelWidth: 500,
+	    pixelHeight: 500
+	};
+
+	var inputCanvas = canvas(inputSettings),
+		outputCanvas = canvas(outputSettings),
+		matrix = [[2,1],[1,-1]],
+		score = 0,
+		inputVector = vector(),
+		oldOutputVector = vector(), 
+		outputVector = vector(),
+		targets = [target(Math.getRandomArbitrary(-9,9), Math.getRandomArbitrary(-9,9))];
+		// change math to our 
+
+	function updateLevel1(event) {
+
+		// get coordinates
+		var x = event.point.x,
+			y = event.point.y;
+
+		// update input/output vectors
+		inputVector.updateVector(x,y);
+		newCoordinates = applyMatrix(x,y,matrix);
+
+		// this should copy the new into the old.
+		oldOutputVector.copy(outputVector);
+		outputVector.updateVector(newCoordinates[0],newCoordinates[1]);
+
+		// draw input/output vectors
+		inputCanvas.drawVector(inputVector);
+		outputCanvas.drawVector(outputVector);
+
+		// check collision (use line segments)
+		var result = outputCanvas.checkCollisions(oldOutputVector,outputVector,targets);
+
+		if(result.length !== 0)
+
+			for(var i = 0; i < result.length; i++) {
+				// change x,y values for target
+				this.randomizeTarget(result[i]);
+				score += 10;
+			}
+		}
+		// calculate proximity to target and update target color
+		var dist = 0;
+
+		for(var i = 0; i < targets.length; i++) {
+			dist = outputCanvas.proximity(outputVector, targets[i]);
+			this.target[i].updateColor(dist);
+		}
+
+		outputCanvas.drawTargets(targets);
+
+	}
+
+	// accepts a single target object as input
+	function randomizeTarget(target) {
+		target.update(Math.getRandomArbitrary(-9,9), Math.getRandomArbitrary(-9,9));
+	}
+},{}],5:[function(require,module,exports){
+/**
+* Level Tracking
+* @description: Mechanism for tracking levels in gameplay
+*/
+
+// Track the levels
+var levelTracking = 1;
+function loadPage(id, levelMove, guide){
```

Comment:
We can start to modify this and the rest of the level tracking. Game one is almost complete.